faf/api/clan_schema.py

```diff
@@ -14,6 +14,7 @@ class ClanSchema(Schema):
     clan_tag = fields.String()
     clan_leader_id = fields.Integer()
     clan_founder_id = fields.Integer()
+    clan_tag_color = fields.String()
```

Comment:
You mean `color` (en_US) right? Grepping through the current codebase returns more `color` than `colour` 

(furthermore, GB should join the rest of the world and start using right-lane traffic, meter-system etc)